src/features/architect/utils/contractHistory/contractEventRecords.ts

```
@@ -0,0 +1,971 @@
+ */
+export const CONTRACT_ROOT_EVENT_KIND: ContractEventKind = 'signing';
+
+export type ContractEventStatus = 'current' | 'superseded';
+
+/** Identity of the world, contract, player, and team an event belongs to. */
+export interface ContractEventSubject {
+  readonly worldId: string;
+  readonly contractId: string;
+  readonly playerId: string;
+  readonly teamId: string;
+}
```

ESLint

[error] Do not declare Player*/Contract* interfaces outside src/schemas. Import from src/schemas/players_v2 instead.

(no-restricted-syntax)